.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+# For pngquant see: https://pngquant.org
+
+
+repos:
+  - repo: https://github.com/gdubicki/pre-commit-pngquant
+    rev: 9fad42f9df2ec8306c73d3e7c0628893ead0ebeb # Use the sha / tag you want to point at
```

Comment:
Any reason this is pinned to this particular version which is not latest?

Reply:
I tried with v3.0.1 and  v2.1.0, but it failed to identify. Thus, I went with a particular version

Reply:
and why that revision instead of the currently latest (9010345cf034fe1097b3b7bb9ecd5ea92dcf85dc)?

Also note that https://github.com/gdubicki/pre-commit-pngquant  does not provide versions/tags, so that is not going to work. You have to use git commit hashes

Reply:
Ahh, I didn't know it doesn't support tags. 

I randomly chose that revision, since I wasn't sure if this is correct thing to do or it should be tag instead. I will update it with the latest